src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -1188,8 +1344,25 @@ def forward(
             is_global_attn=is_global_attn,
             output_attentions=output_attentions,
         )
-        attn_output = self.output(self_outputs[0], hidden_states)
+
+        attn_output = self.self_output(self_outputs[0], hidden_states)
         outputs = (attn_output,) + self_outputs[1:]
+
+
+        if self.cross_modality:
+
+            cross_outputs = self.cross(
+                hidden_states = outputs,
+                attention_mask=attention_mask,
+                layer_head_mask=layer_head_mask,
+                encoder_hidden_states = cross_modality_inputs,
+                encoder_attention_mask = cross_modality_attention_masks,
+                output_attentions=output_attentions,
+            )
+            attn_output = self.cross_output(cross_outputs[0], outputs)
```

Comment:
`outputs` is a tuple here. Should we instead do:
```suggestion
            attn_output = self.cross_output(cross_outputs[0], outputs[0])
```